content/blog/share_openllm_online.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "How to Easily Share Openllm API Online"
+description: "Discover how to deploy and share large language models locally using OpenLLM and expose APIs securely via Pinggy tunneling. Perfect for developers and AI enthusiasts."
+date: 2025-02-04T14:00:00+05:30
+draft: false
+tags: ["OpenLLM", "Pinggy", "Self-Hosted AI", "LLM Deployment", "AI Tunneling"]
+og_image: "images/share_openllm_online/thumbnail_image.png"
+schemahowto: "PHNjcmlwdCB0eXBlPSJhcHBsaWNhdGlvbi9sZCtqc29uIj4KewogICJAY29udGV4dCI6ICJodHRwczovL3NjaGVtYS5vcmcvIiwKICAiQHR5cGUiOiAiSG93VG8iLAogICJuYW1lIjogIlNlbGYtSG9zdCBMYXJnZSBMYW5ndWFnZSBNb2RlbHMgKExMTXMpIHdpdGggT3BlbkxMTSBhbmQgUGluZ2d5IiwKICAiZGVzY3JpcHRpb24iOiAiTGVhcm4gaG93IHRvIGRlcGxveSBMTE1zIGxvY2FsbHkgdXNpbmcgT3BlbkxMTSBhbmQgc2VjdXJlbHkgZXhwb3NlIHRoZW0gb25saW5lIHZpYSBQaW5nZ3kgdHVubmVsaW5nLiIsCiAgImltYWdlIjogImh0dHBzOi8vcGluZ2d5LmlvL2ltYWdlcy9zaGFyZV9vcGVubGxtX29ubGluZS90aHVtYm5haWxfaW1hZ2UucG5nIiwKICAic3RlcCI6IFsKICAgIHsKICAgICAgIkB0eXBlIjogIkhvd1RvU3RlcCIsCiAgICAgICJ0ZXh0IjogIkluc3RhbGwgT3BlbkxMTSBhbmQgZGVwbG95IGEgbW9kZWwuIgogICAgfSwKICAgIHsKICAgICAgIkB0eXBlIjogIkhvd1RvU3RlcCIsCiAgICAgICJ0ZXh0IjogIlN0YXJ0IGEgbW9kZWwgc2VydmVyIHVzaW5nIE9wZW5MTE0sIGUuZy4sICdvcGVubGxtIHNlcnZlIGxsYW1hMy4yOjFiLWluc3RydWN0LWdnbWwtZnAxNi1saW51eCcuIgogICAgfSwKICAgIHsKICAgICAgIkB0eXBlIjogIkhvd1RvU3RlcCIsCiAgICAgICJ0ZXh0IjogIkNyZWF0ZSBhIHNlY3VyZSB0dW5uZWwgd2l0aCBQaW5nZ3k6ICdzc2ggLXAgNDQzIC1SMDpsb2NhbGhvc3Q6MzAwMCBhLnBpbmdneS5pbycuIgogICAgfSwKICAgIHsKICAgICAgIkB0eXBlIjogIkhvd1RvU3RlcCIsCiAgICAgICJ0ZXh0IjogIkFjY2VzcyB5b3VyIExMTSBBUEkgYW5kIE9wZW4gV2ViVUkgdmlhIHRoZSBnZW5lcmF0ZWQgUGluZ2d5IHB1YmxpYyBVUkwuIgogICAgfSwKICAgIHsKICAgICAgIkB0eXBlIjogIkhvd1RvU3RlcCIsCiAgICAgICJ0ZXh0IjogIkVuaGFuY2Ugc2VjdXJpdHkgYnkgYWRkaW5nIGF1dGhlbnRpY2F0aW9uIGFuZCBzZXR0aW5nIHVwIGEgY3VzdG9tIGRvbWFpbi4iCiAgICB9CiAgXQp9Cjwvc2NyaXB0Pg=="
+outputs:
+  - HTML
+  - AMP
+---
+
+In the era of generative AI, self-hosting large language models (LLMs) gives developers full control over data privacy and model customization. {{< link href="https://github.com/bentoml/OpenLLM" >}} OpenLLM{{< /link >}} emerges as a powerful toolkit for deploying {{< link href="https://github.com/bentoml/openllm-models?tab=readme-ov-file#supported-models" >}}models{{< /link >}} like Llama 3 or Mistral locally, while [Pinggy](https://pinggy.io) enables secure internet exposure without complex infrastructure. This guide walks you through creating a production-ready LLM endpoint accessible globally in minutes.
+
+{{< image "share_openllm_online/thumbnail_image.png" "Self-Host LLMs with OpenLLM and Pinggy" >}}
+
+{{% tldr %}}
+
+1. **Install OpenLLM & Deploy a Model**  
+   ```bash
+   pip install openllm
+   openllm serve llama3.2:1b-instruct-ggml-fp16-linux
```

Comment:
below it write Change `llama3.2:1b-instruct-ggml-fp16-linux` to the model you want to execute.